exonum/src/runtime/error.rs

```diff
@@ -526,21 +536,24 @@ impl Display for ExecutionError {
         if let Some(ref call_site) = self.call_site {
             write!(
                 formatter,
-                "Execution error `{kind}` occurred in {site}",
+                "Execution error with code `{kind}` occurred in {site}",
                 kind = self.kind,
                 site = call_site
             )?;
         } else if let Some(runtime_id) = self.runtime_id {
             write!(
                 formatter,
-                "Execution error `{kind}` occurred in runtime {id}",
+                "Execution error with code `{kind}` occurred in {runtime}",
                 kind = self.kind,
-                id = runtime_id
+                runtime = match RuntimeIdentifier::try_from(runtime_id) {
+                    Ok(runtime) => format!("{} runtime", runtime),
```

Comment:
Nit: IMO, it would make more sense to include "runtime" qualifier into `impl Display for RuntimeIdentifier` and simply use `runtime.to_string()` here.